include/malloc.h

```diff
@@ -31,6 +31,8 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
+#define malloc_usable_size malloc_size
+
```

Comment:
@xiaoxiang781216 Why is this needed?

Reply:
> @acassis could you merge this PR?

I'll wait your respond David about the malloc_size and then will merge it.

Reply:
Since the standard doesn't define API to get the malloc size, It isn't unfortunate that popular Unix variant OS define two different function for it:

1. Linux and FreeBSD define malloc_usable_size:
    https://man7.org/linux/man-pages/man3/malloc_usable_size.3.html
    https://www.freebsd.org/cgi/man.cgi?query=malloc_usable_size&apropos=0&sektion=0&manpath=FreeBSD+7.1-RELEASE&format=html
2. macOS define malloc_size:
    https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/malloc_size.3.html

Reply:
But a simple macro can improve the portability without bad side effect.

Reply:
@xiaoxiang781216 - How about adding a comment "For Linux and MacOS compatibility"

Reply:
Done.